crudApp/src/reducers/Crud.js

```diff
@@ -0,0 +1,35 @@
+//A single reducer changeTheState which defines how to perform actions based on the action type.
+
+const initialState = [];
```

Comment:
initial state should have detailed structure e.g if you are having id, name,  address you should mention them in initial state itself. It helps when you try to reset state and removes the ambiguity.